backend/apps/matching/models/participant.py

```diff
@@ -48,6 +49,29 @@ def new(cls, participant_type, email, **kwargs):
     def p_type(self):
         return self.participant_type
 
+    @transaction.atomic
+    def change_approval(self, approver):
```

Comment:
I am unsure whether I like a toggle-button for this. What happens when I press the button twice on slow PCs/internet?

Reply:
nothing weird, the transactions are atomic so there should be no weird stuff hapenning